ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -168,3 +168,7 @@ def fmax(input, other, *, out=None):
         other,
         out=out,
     )
+
+
+def fmin(input, other, *, out=None):
```

Comment:
This function is supposed to handle nans but behaves differently from `torch.fmin` behavior.

Reply:
@hirwa-nshuti Hello, I am going to check it again. Thank you for notifying (:

Reply:
Hi there.
I think I can have a quick fix for this, can I open a PR for the fix?

Reply:
@hirwa-nshuti Oh, you already fixed it. Yeah, please open the PR!